hw3/02-charts.js

```diff
@@ -29,24 +29,71 @@ const borderColors = [
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
 
-const renderChart = () => {
+async function fetchAPI () {
+  try {
+    const response = await fetch(url);
+    const data = await response.json();
+    data.forEach((character) => {
+      console.log(character.family);
+    });
+    return data;
+  }
+  catch (error) {
+    console.log("ERROR MESSAGE: ", error);
+  }
+}
+
+function countCharacterByHouse(characters){
```

Comment:
The [Airbnb guide](https://github.com/airbnb/javascript#functions) recommends function expressons rather than function delarations.